### PR TITLE
Add SCPSL Exiled and update SCPSL Egg to use yolks image

### PR DESCRIPTION
**Pull request by gOOvER**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### New Server Submissions:

1. [x] Does your submission pass tests (server is connectable)?
2. [ ] Does your server use a custom docker image?
    * [ ] Have you tried to use a generic image?
    * [ ] Did you PR the necessary changes to make it work?
3. [x] Have you added the server to the main README.md?
4. [x] Have you added a unique README.md for the server you are adding?

### Changes to an existing Egg:

1. [x] Have you added an explanation of what your changes do and why you'd like us to include them?
2. [x] Have you tested your Egg changes?

### CODE REVIEW

**Comment by gOOvER** [comment]
Log part of install:

```
[2021-11-14 11:06:07.275 +01:00] [Exiled.Bootstrap] Exiled is loading...
[2021-11-14 11:06:07.285 +01:00] [INFO] [Exiled.Loader] Initializing at /home/container
[2021-11-14 11:06:07.298 +01:00] Exiled.Loader - Version 3.6.2
[2021-11-14 11:06:07.310 +01:00] [INFO] [Exiled.Loader] Loading dependencies at /home/container/.config/EXILED/Plugins/dependencies
[2021-11-14 11:06:07.322 +01:00] [INFO] [Exiled.Loader] Loaded dependency 0Harmony@2.0.4
[2021-11-14 11:06:07.334 +01:00] [INFO] [Exiled.Loader] Loaded dependency Exiled.API@3.6.2
[2021-11-14 11:06:07.347 +01:00] [INFO] [Exiled.Loader] Loaded dependency SemVer@1.3.0
[2021-11-14 11:06:07.359 +01:00] [INFO] [Exiled.Loader] Loaded dependency YamlDotNet@9.0.0
[2021-11-14 11:06:07.370 +01:00] [INFO] [Exiled.Loader] Dependencies loaded successfully!
[2021-11-14 11:06:07.382 +01:00] [INFO] [Exiled.Loader] Loaded plugin Exiled.CreditTags@3.6.2
[2021-11-14 11:06:07.395 +01:00] [INFO] [Exiled.Loader] Loaded plugin Exiled.CustomItems@3.6.2
[2021-11-14 11:06:07.407 +01:00] [INFO] [Exiled.Loader] Loaded plugin Exiled.CustomRoles@3.6.2
[2021-11-14 11:06:07.417 +01:00] [INFO] [Exiled.Loader] Loaded plugin Exiled.Events@3.6.2
[2021-11-14 11:06:07.429 +01:00] [INFO] [Exiled.Loader] Loaded plugin Exiled.Permissions@3.6.2
[2021-11-14 11:06:07.442 +01:00] [INFO] [Exiled.Loader] Loaded plugin Exiled.Updater@3.1.1
[2021-11-14 11:06:07.454 +01:00] [INFO] [Exiled.Loader] Loading plugin configs...
[2021-11-14 11:06:07.466 +01:00] [INFO] [Exiled.Loader] Plugin configs loaded successfully!
[2021-11-14 11:06:07.479 +01:00] [INFO] [Exiled.Loader] Loading plugin translations...
[2021-11-14 11:06:07.491 +01:00] [INFO] [Exiled.Loader] Plugin translations loaded successfully!
[2021-11-14 11:06:07.501 +01:00] [INFO] [Exiled.API] Exiled.Events v3.6.2 by Exiled Team has been enabled!
[2021-11-14 11:06:07.886 +01:00] Loading gameplay config...
[2021-11-14 11:06:07.899 +01:00] Processing rate limits...
[2021-11-14 11:06:07.910 +01:00] Rate limiting loaded
[2021-11-14 11:06:07.922 +01:00] Loading sharing config...
[2021-11-14 11:06:07.935 +01:00] Processing shares...
[2021-11-14 11:06:07.947 +01:00] Config sharing loaded.
[2021-11-14 11:06:07.958 +01:00] Validating bans...
[2021-11-14 11:06:07.970 +01:00] Bans has been validated.
[2021-11-14 11:06:07.982 +01:00] Whitelist has been loaded.
[2021-11-14 11:06:07.994 +01:00] Whitelist has been loaded.
[2021-11-14 11:06:08.006 +01:00] Loading saved mutes...
[2021-11-14 11:06:08.019 +01:00] Can't load the mute file!
[2021-11-14 11:06:08.031 +01:00] Reserved slots list has been loaded.
[2021-11-14 11:06:08.043 +01:00] Reserved slots list has been loaded.
[2021-11-14 11:06:08.053 +01:00] Server tickrate set to: 60
[2021-11-14 11:06:08.066 +01:00] Config sharing loaded.
[2021-11-14 11:06:08.078 +01:00] Loading saved mutes...
[2021-11-14 11:06:08.090 +01:00] Can't load the mute file!
[2021-11-14 11:06:08.762 +01:00] [INFO] [Exiled.Events] Patching completed in 00:00:01.3632064
[2021-11-14 11:06:08.787 +01:00] [INFO] [Exiled.API] Exiled.CustomItems v3.6.2 by Exiled Team has been enabled!
[2021-11-14 11:06:08.798 +01:00] [INFO] [Exiled.API] Exiled.CustomRoles v3.6.2 by Exiled Team has been enabled!
[2021-11-14 11:06:08.810 +01:00] [INFO] [Exiled.API] Exiled.CreditTags v3.6.2 by Babyboucher20 & iRebbok & Exiled Team has been enabled!
[2021-11-14 11:06:08.823 +01:00] [INFO] [Exiled.API] Exiled.Updater v3.1.1 by Exiled.Updater has been enabled!
[2021-11-14 11:06:08.835 +01:00] [INFO] [Exiled.API] Exiled.Permissions v3.6.2 by Exiled Team has been enabled!
[2021-11-14 11:06:08.847 +01:00] Welcome to
   ▄████████ ▀████    ▐████▀  ▄█   ▄█          ▄████████ ████████▄
  ███    ███   ███▌   ████▀  ███  ███         ███    ███ ███   ▀███
  ███    █▀     ███  ▐███    ███▌ ███         ███    █▀  ███    ███
 ▄███▄▄▄        ▀███▄███▀    ███▌ ███        ▄███▄▄▄     ███    ███
▀▀███▀▀▀        ████▀██▄     ███▌ ███       ▀▀███▀▀▀     ███    ███
  ███    █▄    ▐███  ▀███    ███  ███         ███    █▄  ███    ███
  ███    ███  ▄███     ███▄  ███  ███▌    ▄   ███    ███ ███   ▄███
  ██████████ ████       ███▄ █▀   █████▄▄██   ██████████ ████████▀
                                  ▀
[2021-11-14 11:06:08.954 +01:00] Loading configs...
[2021-11-14 11:06:08.966 +01:00] Loading gameplay config...
[2021-11-14 11:06:08.979 +01:00] Processing rate limits...
[2021-11-14 11:06:08.991 +01:00] Rate limiting loaded
[2021-11-14 11:06:09.002 +01:00] Loading sharing config...
[2021-11-14 11:06:09.014 +01:00] Processing shares...
[2021-11-14 11:06:09.026 +01:00] Config sharing loaded.
[2021-11-14 11:06:09.039 +01:00] Validating bans...
[2021-11-14 11:06:09.051 +01:00] Bans has been validated.
[2021-11-14 11:06:09.061 +01:00] Whitelist has been loaded.
[2021-11-14 11:06:09.074 +01:00] Loading saved mutes...
[2021-11-14 11:06:09.087 +01:00] Can't load the mute file!
[2021-11-14 11:06:09.099 +01:00] Reserved slots list has been loaded.
[2021-11-14 11:06:09.109 +01:00] Server tickrate set to: 60
[2021-11-14 11:06:09.122 +01:00] Config files loaded from /home/container/.config/SCP Secret Laboratory/config/10101/
[2021-11-14 11:06:09.134 +01:00] Game version: 11.0.0
[2021-11-14 11:06:09.146 +01:00] Scene Manager: Loaded scene 'FastMenu' [Assets/_Scenes/FastMenu.unity]
[2021-11-14 11:06:09.159 +01:00] Online mode is ENABLED.
[2021-11-14 11:06:09.171 +01:00] Query port disabled in config!
[2021-11-14 11:06:09.181 +01:00] Starting server...
[2021-11-14 11:06:09.193 +01:00] Obtaining your external IP address...
[2021-11-14 11:06:09.206 +01:00] [INFO] [Exiled.Updater] Found the smallest version of Exiled - Exiled.Loader:3.6.2
```

File: game_eggs/steamcmd_servers/scpsl/exiled/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+### Verification
+
+The server must be verified with the SCP:SL developers in order for it to be visible in the server browser.
+
+Instructions are as follows:
+* Make sure your server complies with the [Verified Server Rules](https://scpslgame.com/Verified_server_rules.pdf).
+* Send an email to `server.verification@scpslgame.com` from your configured contact email with the following information:
+  1. The **public** IPv4 address of your server, with port. e.g. `69.69.69.69:7777`
+  2. If the server's IP is static or dynamic.
```
Comment (Software-Noob) [inline game_eggs/steamcmd_servers/scpsl/exiled/README.md:24]: This looks like an unfinished sentence?

File: game_eggs/steamcmd_servers/scpsl/exiled/egg-s-c-p--s-l--exiled.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    "images": [
+        "ghcr.io\/parkervcp\/yolks:mono_latest"
+    ],
+    "file_denylist": [],
+    "startup": ".\/LocalAdmin {{SERVER_PORT}}",
+    "config": {
+        "files": "{}",
+        "startup": "{\r\n    \"done\": \"Waiting for players..\"\r\n}",
+        "logs": "{\r\n    \"custom\": false,\r\n    \"location\": \"logs\/latest.log\"\r\n}",
+        "stop": "^C"
```
Comment (Software-Noob) [inline game_eggs/steamcmd_servers/scpsl/exiled/egg-s-c-p--s-l--exiled.json:21]: This game has its own stop command, `EXIT` that would gracefully stop it

**Review by Software-Noob — CHANGES_REQUESTED** [review]

File: game_eggs/steamcmd_servers/scpsl/exiled/egg-s-c-p--s-l--exiled.json (hunk 2 above)
Comment (gOOvER) [reply game_eggs/steamcmd_servers/scpsl/exiled/egg-s-c-p--s-l--exiled.json]: done

File: game_eggs/steamcmd_servers/scpsl/exiled/README.md (hunk 1 above)
Comment (gOOvER) [reply game_eggs/steamcmd_servers/scpsl/exiled/README.md:24]: looks correct. but i never add this. the readme is c&p from dedicated

**Review by Software-Noob — APPROVED** [review]
LGTM, thank you for contributing

**Pull request merged**